pam/pam_adsys.c

```diff
@@ -223,7 +224,16 @@ static char *get_default_sss_domain(pam_handle_t *pamh) {
                 pam_syslog(pamh, LOG_ERR, "Could not find value for key 'default_domain_suffix' in sssd.conf");
                 break;
             }
-            while (*++domain == ' ') continue;
+            // Ignores whitespaces and tabs right after the '='
+            domain++;
+            while (strlen(domain) > 0 && (*domain == ' ' || *domain == '\t')) domain++;
+
+            if (strlen(domain) == 0) {
+                pam_syslog(pamh, LOG_ERR, "Could not find value for key 'default_domain_suffix' in sssd.conf");
+                domain = NULL;
+                break;
+            }
+
```

Comment:
As @CarlosNihelton pointed out, we weren't considering cases when sssd.conf had something like `default_domain_suffix = \t domain.com`, so it could create issues when parsing it. I also added a check for cases where the config has something like `default_domain_suffix =         \n`.